java-checks-test-sources/src/main/java/checks/MembersDifferOnlyByCapitalizationCheck.java

```diff
@@ -1,35 +1,40 @@
-class A implements MyInterface {
+package checks;
+
+class MembersDifferOnlyByCapitalizatuionCheck {
+
+abstract class A implements MyInterface {
   private static final long var1 = -9215047833775013803L; // Compliant
   public long var2 = 0L;
-  
+
   public static void equAls(Object obj) {} // Noncompliant [[sc=22;ec=28]] {{Rename method "equAls" to prevent any misunderstanding/clash with method "equals" defined in superclass "java.lang.Object".}}
```

Comment:
This code does not compile on my side: "inner class can not have static declarations".

Reply:
that's strange. it compile on my side.

Reply:
removing the static modifier